.github/workflows/publish-documentation-gh-pages.yml

```diff
@@ -1,6 +1,8 @@
 name: Publish Documentation to GitHub Pages
 
 on:
+  merge_group:
+    types: [checks_requested]  # when a pull request is added to a merge queue
```

Comment:
I think that is not needed.
```suggestion
    branches: [main]  # when a pull request is added to a merge queue
```

Reply:
I think the branches argument might be needed since we only want to publish when merge targets main. Alternatively we could just use push?

Reply:
yeah, seems like push is the simpler and more sensible solution for us.